plugins/newsdownloader.koplugin/epubdownloadbackend.lua

```diff
@@ -29,11 +42,27 @@ local FeedCache = CacheSQLite:new{
     size = 1024 * 1024 * 10, -- 10MB
 }
 
+-- get HTML elements sorted by level
```

Comment:
Total nit, but that looks better in an IDE as a proper sentence (when hovering over the function where it's used) and that's our general comment style.
```suggestion
-- Get HTML elements sorted by level.
```